cloudflare/package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "scripts": {
     "deploy": "npx wrangler deploy --minify",
-    "publish": "npx wrangler publish --minify",
```

Comment:
publish is deprecated https://developers.cloudflare.com/workers/wrangler/commands/#publish

Reply:
We should add `npm run deploy` so that there is only one command we work with.

Reply:
it's right above it